source/index.d.ts

```diff
@@ -8,6 +8,7 @@ export class MaxBufferError extends Error {
 
 // eslint-disable-next-line @typescript-eslint/ban-types
 type TextStreamItem = string | Buffer | ArrayBuffer | ArrayBufferView;
+// eslint-disable-next-line n/no-unsupported-features/node-builtins
 export type AnyStream<SteamItem = TextStreamItem> = Readable | ReadableStream<SteamItem> | AsyncIterable<SteamItem>;
```

Comment:
`ReadableStream` is supported in Node 18 but is marked as experimental.
However, it is a web standard, and the Node.js implementation has been pretty stable for a few years now.